app/views/recipe_foods/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1 class="mt-5">New recipe food</h1>
+
+<%= render "form", recipe_food: @recipe_food %>
+
+<br>
+
+<div>
+  <%= link_to "Back to recipe foods", recipe_foods_path %>
+</div>
```

Comment:
I have tried several ways and times to create a new recipe food or ingredient but I always meet this error shown in the picture below :point_down: meanwhile the food and recipe has been created already. :confused:  
To avoid this kindly fetch the data from the recipe and food that has been created already and display them in a dropdown. The user will then select the ones he/she wants to use in the food and recipe inputs. 

![Screenshot from 2023-06-30 22-12-36](https://github.com/veronica365/recepies-app/assets/65233860/79da22f3-9dcf-49c1-bc2c-422901227550)

Reply:
Thanks for the review . Drop downs have now been added